dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -389,6 +402,7 @@ function spawn_pirate( param )
             can_capture = false
          end
       else
+         misn.appendLog("No sign of pirate in expected location, you were too late")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.